crates/sui-core/src/authority/authority_store.rs

```diff
@@ -1418,7 +1418,7 @@ impl<T: ModuleResolver> ModuleResolver for ResolverWrapper<T> {
 // The primary key type for object storage.
 #[serde_as]
 #[derive(Eq, PartialEq, Clone, Copy, PartialOrd, Ord, Hash, Serialize, Deserialize, Debug)]
-pub(crate) struct ObjectKey(pub ObjectID, pub VersionNumber);
+pub struct ObjectKey(pub ObjectID, pub VersionNumber);
```

Comment:
Curious why this needed to change?

Reply:
> Curious why this needed to change?

Without this, we will not be able to dump the DB in external utils as this will leak the private type.